optimizely/notification/manager.go

```diff
@@ -24,6 +24,7 @@ import (
 // Manager is a generic interface for managing notifications of a particular type
 type Manager interface {
 	AddHandler(func(interface{})) (int, error)
+	RemoveHandler(id int)
```

Comment:
Please add unit tests for this method in the manager

Reply:
Thanks for pointing out, done!